RELEASE_NOTE.md

```diff
@@ -1,5 +1,132 @@
 ## Release Notes of the Log Layout Based HDF5 Virtual Object Layer
 
+### Version 1.4.0 (Dec 05, 2022)
+* New features
+  + Support fill mode. See commit 8aee024
+  + Support NetCDF4. By setting the two VOL environment variables
+    `HDF5_VOL_CONNECTOR` and `HDF5_PLUGIN_PATH`, NetCDF4 programs now can write
+    data to files in log layout through Log VOL. See PR #15.
+  + Support using Log VOL as a Pass-through VOL.
+    * This feature adds the support of opening and operating an existing
+      regular HDF5 file.
+    * This feature adds the option of performing all writes using the
+      underlying VOL.
+    * When the Pass-through VOL is enabled, it uses collective MPI I/O to
+      perform file writes. See PR #42.
+    * For its usage, refer to the User Guide in doc/usage.md.
+    * See PR #33.
+  + Support VOL connector interface version 3.
+    * HDF5 1.13.3 requires a version 3 connector.
+    * Log VOL will expose the version 3 interface when the HDF5 version 
```

Comment:
What is "version 3 interface"? 
any URL reference to HDF5 documents?

Reply:
HDF5 1.13.3 uses a different VOL callback interface to accommodate multi-dataset API.
I don't think there will be any document until 1.14 release.